downstream/modules/platform/proc-configure-upgraded-aap.adoc

```diff
@@ -0,0 +1,15 @@
+[id="proc-configure-upgraded-aap_{context}"]
+
+= Configuring your upgraded {PlatformNameShort}
+
+== Configuring {ControllerName} instance groups
+
+After upgrading your {PlatformName} instance, associate your original instances to its corresponding instance groups by configuring settings in the {ControllerName} UI:
+
+.Procedure
+. Log in to new Controller instance
+. Content from old instance, such as credentials, jobs, inventories should now be visible on your Controller instance
+. Navigate to *Administration* > *Instance Groups*
+. Associate execution nodes by clicking on an instance group, then click the Instances tab
```

Comment:
I would bold **Instances** because it is an actual tab name and something you click on.